src/components/Tabs/Annotation/CreateLabelForm.tsx

```diff
@@ -18,32 +19,46 @@ type CreateLabelFormProps = {
   confirmText?: string;
   focusNameInput?: boolean;
   zIndex?: number;
+  allowTypeSelection?: boolean;
```

Comment:
Type selection is only enabled at label creation, and is disabled when editing labels after the fact.